src/lib/forms/rjsf/fields/RJSFLabelField.js

```diff
@@ -0,0 +1,35 @@
+import HiddenWidget from '@rjsf/core/dist/cjs/components/widgets/HiddenWidget';
+import _capitalize from 'lodash/capitalize';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Header } from 'semantic-ui-react';
+
+/**
+ * Custom field to display a field as a label and the value in the hidden field
```

Comment:
I am not sure if I understand how this field should behave, maybe we could see it in a call

Reply:
Yes, in a couple of words, if you have a readonly input the text is grayed out, almost unreadable.
The idea is to display it a label (see the name of the days in the screenshot Monday, ...) but it has to be part of the form and sent to the backend so it adds an hidden field with as value the name of the day.